site/profile/manifests/ccs/daq_client.pp

```diff
@@ -0,0 +1,77 @@
+## @summary
+##   Installation of DAQ client:
+##   Download DAQ tarfile, extract it, create /etc/ccs config files.
+
+class profile::ccs::daq_client (
```

Comment:
This appears to be completely stand alone and could be in an external module.  There are a bunch of example of things in the tree that should be in an external module as they aren't a composition.  I don't object to merging this now but it should probably be moved external at some point in the future in the interest of keeping our profile tree small.

Reply:
Thanks, I will try to separate it out at some point in the future.